src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/body-open.html.twig

```diff
@@ -1 +1 @@
-<noscript><p><img src="{{ analytics.content.url }}/piwik.php?idsite={{ analytics.content.siteId }}" style="border:0;" alt="" /></p></noscript>
+<noscript><p><img referrerpolicy="no-referrer-when-downgrade" src="{{ analytics.content.url }}/matomo.php?idsite={{ analytics.content.siteId }}&amp;rec=1" style="border:0;" alt="" /></p></noscript>
```

Comment:
This also needs the `|trim('/')` filter to ensure there is not a duplicate forward slash.

Reply:
You're right. This seems to be an issue even before my changes.